common/src/main/java/com/radixdlt/monitoring/Metrics.java

```diff
@@ -372,5 +373,9 @@ public enum Direction {
     }
   }
 
-  public record Config(String branchAndCommit, String key, HashCode postGenesisEpochStateHash) {}
+  public record Config(
+      String branchAndCommit,
+      String key,
+      @Nullable String componentAddress, // null when not configured as validator
```

Comment:
Can we call this validatorAddress instead?

Reply:
Is this the statically configured (or genesis-resolved) validator address?

IE presence of this address doesn't actually apply the node is currently validating, right? 

Perhaps we should have another boolean metric for that, to help validators planning node maintenance?

Reply:
> Is this the statically configured (or genesis-resolved) validator address?

Yes.

> Perhaps we should call this configuredValidatorAddress?

Ok, done 👍 

> Perhaps we should have another boolean metric for "currently validating in epoch" under the consensus section, to help validators planning node maintenance? (if we don't already)

Yup, we have it as `rn_bft_in_validator_set`: https://github.com/radixdlt/babylon-node/blob/develop/common/src/main/java/com/radixdlt/monitoring/Metrics.java#L181